kotlin/samples/dungeon/common/src/main/java/com/squareup/sample/dungeon/board/Board.kt

```diff
@@ -15,7 +15,25 @@
  */
 package com.squareup.sample.dungeon.board
 
+import kotlinx.serialization.Serializable
+
+/**
+ * Information about the board, encoded as YAML at the start of a board file.
+ *
+ * The metadata must be surrounded by lines containing only "`---`".
+ *
+ * @see parseBoardMetadata
+ */
+@Serializable
```

Comment:
I saw a Snapshot.EMPTY in here somewhere, sure you're actually using this?

Reply:
Yes, see `BoardMetadata.serializer()` in `parseBoardMetadata`.